Demo/Tweakable/Device.swift

```diff
@@ -0,0 +1,136 @@
+import UIKit
+
+struct Device {
+    enum `Type`: String {
+        case phone4inch = "iPhone 5S (4-inch)"
+
+        // swiftlint:disable:next identifier_name
+        case phone4_7inch = "iPhone 8 (4.7-inch)"
+
+        // swiftlint:disable:next identifier_name
+        case phone5_5inch = "iPhone 8 Plus (5.5-inch)"
+
+        // swiftlint:disable:next identifier_name
+        case phone5_8inch = "iPhone X (5.8-inch)"
```

Comment:
Do we also need 6.1-inch iPhone11?

Reply:
I thought about it but the aspect ratio it's not very different from the iPhone X.